app/controllers/application_controller.rb

```diff
@@ -13,13 +13,13 @@ def require_user
       redirect_to login_path
     end
   end
-  
+
   def current_admin?
     current_user && current_user.admin?
   end
 
   def current_user
-    @current_user ||= User.find(session[:user_id]) if session[:user_id]
+    @current_user ||= User.find(session[:user_id]) if session[:user_id] && User.exists?(id: session[:user_id])
```

Comment:
I dont think this is needed. Theres users in the database. There were none in your development db and that's the reason it was failing. There are users on the production db so this addition is not needed imo